app/views/govuk_publishing_components/components/_summary_card.html.erb

```diff
@@ -0,0 +1,58 @@
+<%
+  add_gem_component_stylesheet("summary-card")
+
+  id ||= nil
+  title ||= nil
+  data_attributes ||= {}
+  summary_card_actions ||= []
+  rows ||=[]
+%>
+<% if title || rows.any? %>
+  <%= tag.div class: "gem-c-summary-card", id: id, data: data_attributes do %>
+    <%= tag.div class: "govuk-summary-card" do %>
+      <%= tag.div class: "govuk-summary-card__title-wrapper" do %>
+        <%= tag.h2 class: "govuk-summary-card__title" do %>
+          <%= title %>
+        <% end %>
+
+        <%= tag.ul class: "govuk-summary-card__actions" do %>
+          <% summary_card_actions.each do |action| %>
+            <%= tag.li class: "govuk-summary-card__action" do %>
+              <%= link_to sanitize(action[:label] + tag.span(" #{title}", class: "govuk-visually-hidden")), action[:href], class: "govuk-link govuk-link--no-visited-state #{"gem-link--destructive govuk-!-font-weight-bold" if action[:destructive]}".strip %>
+            <% end %>
+          <% end %>
+        <% end %>
+      <% end %>
+
+      <% if rows.present? %>
+        <%= tag.div class: "govuk-summary-card__content" do %>
+          <%= tag.dl class: "govuk-summary-list" do %>
+            <% rows.each do |row| %>
+              <%= tag.div class: "govuk-summary-list__row" do %>
+                <%= tag.dt class: "govuk-summary-list__key" do %>
+                  <%= row[:key] %>
+                <% end %>
+
+                <%= tag.dt class: "govuk-summary-list__value" do %>
+                  <%= row[:value] %>
+                <% end %>
+
+                <% if row[:actions].present? %>
+                  <%= tag.dd class: "govuk-summary-list__actions" do %>
+                    <% row[:actions].each do |action| %>
+                      <% if action[:opens_in_new_tab] %>
+                        <%= link_to sanitize(action[:label] + tag.span(" #{row[:key]} (opens in new tab)", class: "govuk-visually-hidden")), action[:href], class: "govuk-link govuk-link--no-visited-state govuk-!-margin-left-2", rel: "noreferrer noopener", target: "_blank" %>
+                      <% else %>
+                        <%= link_to sanitize(action[:label] + tag.span(" #{row[:key]}", class: "govuk-visually-hidden")), action[:href], class: "govuk-link govuk-link--no-visited-state govuk-!-margin-left-2 #{"gem-link--destructive" if action[:destructive]}".strip %>
+                      <% end %>
```

Comment:
I'm just wondering if these actions should be pipe separated?

![components-gem-pr-4182 herokuapp com_component-guide_summary_card](https://github.com/user-attachments/assets/315dca88-cc1d-46f1-8938-d92fd6f74d42)

Reply:
Yes, they display like that when used in the title section so replicating that here makes sense. I've just added the `govuk-summary-card__action` class to the links that adds that.

Reply:
Great thanks. Although, I think you can actually now remove these styles:
```
  .govuk-summary-card__action {
    &:not(:first-child):not(:last-child) {
      padding-left: govuk-spacing(2);
    }
  }
```
The `govuk-summary-card__action` seems to add the correct spacing between items (otherwise, there's a bit too much spacing on the middle item)

<img src="https://github.com/user-attachments/assets/0e50fb03-2c30-4e55-99be-250c8641c656" width="400">

Reply:
Great, that looks good to me. Just one last thing: I noticed that on Desktop, when all three actions are displayed, the third one wraps to the next line. This happens because the Design System code sets a fixed width of 20% on the actions column. You could set the cell width to `auto` to keep them all in line:
```
.govuk-summary-list__actions {
  width: auto;
}
```

Reply:
Ah yes, nice one. Have done that. 👍